lib/rubocop/extension/generator/cli.rb

```diff
@@ -4,6 +4,8 @@ module Generator
       class CLI
         BANNER = <<~TEXT
           Usage: rubocop-extension-generator NAME
+
+          The NAME must starts with rubocop-, like rubocop-rspec.
```

Comment:
"start"

Reply:
Good catch! I fixed and squashed. Thanks!